src/current/releases/cloud.md

```diff
@@ -14,6 +14,23 @@ Get future release notes emailed to you:
 
 {% include marketo.html formId=1083 %}
 
+## August 12, 2024
+
+<h3 id="2024-08-12-general-updates"> General updates </h3>
+
+- CockroachDB v24.2 is now generally available for CockroachDB {{ site.data.products.dedicated }}, and is scheduled to be made available for CockroachDB {{ site.data.products.core }} on August 19, 2024{% comment %}Verify{% endcomment %}. CockroachDB v24.2 is an optional [Innovation release]({% link releases/release-support-policy.md %}#support-types). Refer to [Create a CockroachDB {{ site.data.products.dedicated }} cluster]({% link cockroachcloud/create-your-cluster.md %}) or [Upgrade to v24.2]({% link cockroachcloud/upgrade-to-v24.2 %}).
```

Comment:
is it usual for us to say in the cloud release notes when it is scheduled to be available for binary downloads?  My personal opinion is we should NOT do this since this is the cloud release notes for cloud customers 

If self-hosted binary customers want to know when the binary is dropping there are other channels for that info, not this doc

Reply:
That's true. I did it the same way we did for v24.1 and v23.2. I'll remove it. Also I think this date should be 8/13.

Reply:
AIUI we're merging stuff today 8/12 but will confirm whether the date should be 8/12 or 8/13